docs/pages/enroll-resources/desktop-access/active-directory.mdx

```diff
@@ -599,9 +599,17 @@ To configure Teleport to protect access to Windows desktops:
    windows_desktop_service:
      enabled: true
      ldap:
-       # Port must be included for the addr.
+       # Address (including port) of an LDAP server.
+       # Can be omitted if `locate_server` is enabled.
        # LDAPS port is 636 by default (example.com:636)
-       addr: "$LDAP_SERVER_ADDRESS"
+       addr: "$LDAP_SERVER_ADDRESS" 
```

Comment:
Personally, I'd omit `locate_server` from this section. We're not trying to cover every possible configuration option here (this is just step 1), but instead trying to give the reader a starting point.